meta-ostro/conf/distro/ostro.conf

```diff
@@ -12,6 +12,52 @@ SDK_UPDATE_URL = "\
 		${MACHINE}\
 "
 
+# When building in a CI system with swupd enabled, OS_VERSION must be
+# set to a consistent value for all builds. See ostroproject-ci.inc
+# for an example how that works with Jenkins.
+#
+# In local builds, we only have ${DATETIME} as something that
+# increments automatically, but it is too large for an integer number
+# on 32 bit systems. Therefore we substract the 2016 as the initial
+# year in which Ostro OS started using swupd (assumes a 64 build host,
+# to avoid integer overflows in Python itself) and ignore the
```

Comment:
there are no integer overflows in python, it will auto-grow to large number, which can contain more than simple 32/64 bits :-)